x86_64-unknown-intermezzos-gnu.json

```diff
@@ -3,15 +3,20 @@
 	"cpu": "x86-64",
 	"data-layout": "e-m:e-i64:64-f80:128-n8:16:32:64-S128",
 	"executables": true,
-        "linker-flavor": "gcc",
+	"linker": "gcc",
```

Comment:
Will this work on systems where the linker is set in `~/.cargo/config` (e.g. using a cross-dev toolchain on macOS), or when the linker is named something other than `gcc` (e.g. `x86_64-pc-elf-gcc`)?